notebooks/00_quick_start/rlrmc_movielens.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
<p>I think it would be interesting for the user to have a brief description of the algo</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/729/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I will add a brief description of the algo.</p>

Reply:
done.